keyboard/src/lib/y2country/widgets.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may find
+# current contact information at www.novell.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm/widget"
+
+Yast.import "Keyboard"
+
+module Y2Country
```

Comment:
Sorry, I don't remember the naming scheme we have discussed in summer. Have you consulted it? (If yes, where can find it?)
In Users you have used [`module Users`](https://github.com/yast/yast-users/pull/127/files#diff-fa2036669a7ab51839a0ddf0e0b39295R32) which is inconsistent.